content/en/docs/contribute/style/style-guide.md

```diff
@@ -110,6 +110,12 @@ document, use the backtick (`).
   <tr><th>Do</th><th>Don't</th></tr>
   <tr><td>The <code>kubectl run</code> command creates a Deployment.</td><td>The "kubectl run" command creates a Deployment.</td></tr>
   <tr><td>For declarative management, use <code>kubectl apply</code>.</td><td>For declarative management, use "kubectl apply".</td></tr>
+  <tr><td>For bash code use <code>shell</code>.</td><td>For bash use <code>bash</code> or <code>sh</code>.</td></tr>
```

Comment:
This seems confusing. What should I use for code written for a POSIX shell? What about for code that specifically expects `bash`?

Reply:
>What about for code that specifically expects `bash`?

do you have an example for that ?